src/components/Modals/EditPriceAndBeneficiaryModal/EditPriceAndBeneficiaryModal.tsx

```diff
@@ -129,12 +129,12 @@ export default class EditPriceAndBeneficiaryModal extends React.PureComponent<Pr
   }
 
   render() {
-    const { name, isLoading, onClose, onSkip } = this.props
+    const { name, isLoading, mountNode, onClose, onSkip } = this.props
     const { isFree, isOwnerBeneficiary, price = '' } = this.state
     const beneficiary = this.getBeneficiary()
 
     return (
-      <Modal name={name} size="tiny" onClose={onClose}>
+      <Modal name={name} size="tiny" onClose={onClose} mountNode={mountNode}>
```

Comment:
Just out of curiosity, why do we need the `mountNode` now? Is it related to the main issue or is it solving another thing?

Reply:
it's used to solve #2228. Having this `mountNode`, it's easier to know when more than one modal is open and apply styles.